perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -705,7 +705,20 @@ sub processed_summary_string {
 
 sub duration_string {
     my ( $problem, $c ) = @_;
-    my $body = $problem->body( $c );
+    my $body;
+    my $handler = $problem->get_cobrand_body_handler();
+    # If the report was sent to a cobrand that we're not currently on,
+    # include a link to view it on the responsible cobrand.
+    # This only occurs if the report was sent to a single body and we're not already
+    # using the body name as a link to all problem reports.
+    my $bodies_str_ids = $problem->bodies_str_ids;
+    if ( !mySociety::Config::get('AREA_LINKS_FROM_PROBLEMS') && scalar(@$bodies_str_ids) == 1 &&
+         $handler->is_council && $handler->moniker ne $c->cobrand->moniker ){
```

Comment:
Given the is_council, I think it'd be preferable if this could be a function in UKCouncil.pm with a cobrand->can check here to call it.